symfony/templates/help/faq.html.twig

```diff
@@ -9,8 +9,7 @@
     <div class="page-header">
         <h2><i class="fa fa-question-circle-o" aria-hidden="true"></i> Technical FAQs</h2>
     </div>
-    <div class="text-right"><a class="pull-right" href="{{ asset('HealthPro Frequently Asked Questions.pdf') }}" target="_blank"><i class="fa fa-external-link" aria-hidden="true"></i> Open/Download</a></div>
-    <iframe class="pdf" name="pdf" src="{{ asset('HealthPro Frequently Asked Questions.pdf') }}#view=FitH"></iframe>
+    <a href="{{ faqs.web_link }}" target="_blank" class="btn btn-primary"><i class="fa fa-external-link" aria-hidden="true"></i> Click here to visit the HealthPro FAQ document</a>
```

Comment:
There might have been some discussion that I missed, but why do we need this page at all? Why not link directly to the Google Doc in the navigation and help home page like the confluence links?